epictrack-web/src/components/work/WorkForm/index.tsx

```diff
@@ -35,6 +35,7 @@ const schema = yup.object<Work>().shape({
   federal_involvement_id: yup
     .number()
     .required("Federal Involvement is required"),
+  report_description: yup.string().required("Description is required"),

```

Comment:
can you mention the validation message as "Work description is required"

Reply:
sure thing